src/test/java/gov/cabinetofice/gapuserservice/service/user/OneLoginUserServiceTest.java

```diff
@@ -100,8 +101,9 @@ void shouldReturnUserWhenValidSubIsGiven() {
             when(userRepository.findBySub("1234")).thenReturn(Optional.of(mockedUser));
             User result = oneLoginUserService.getUserByUserSub("1234");
 
-            assertThat(result).isNotNull();
-            assertThat(result).isEqualTo(mockedUser);
+            assertThat(result)
+                .isNotNull()
+                .isEqualTo(mockedUser);
```

Comment:
Since `mockedUser` is not null, this assertion is redundant:

```suggestion
            assertThat(result)
                .isEqualTo(mockedUser);
```